alephant-renderer.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake-rspec"
 
   spec.add_runtime_dependency 'i18n'
-  spec.add_runtime_dependency 'mustache', '>= 0.99.5'
+  spec.add_runtime_dependency 'mustache', '~> 0.99.5'
```

Comment:
`< 1` would allow less than `0.99.5` which would violate the previous restriction - this maintains that restriction but keeps it below the next major version.  Equivalent to:
```ruby
spec.add_runtime_dependency 'mustache', '>= 0.99.5', '< 1'
```